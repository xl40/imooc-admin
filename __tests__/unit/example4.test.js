import { shallowMount } from '@vue/test-utils'
import HelloWorld from '@/components/HelloWorld.vue'

describe('test with shallowMount', function() {
  it('组件渲染正常', function() {
    const wrapper = shallowMount(HelloWorld, {
      propsData: { msg: 'Alex' }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
