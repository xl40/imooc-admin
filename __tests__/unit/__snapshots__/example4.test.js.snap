// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test with shallowMount 组件渲染正常 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="hello"
  >
    <h1>
      Alex
    </h1>
  </div>
  <div
    class="test"
  >
     abc 
  </div>
  
</div>
`;
